queries/nvim/highlights.scm

```diff
@@ -1,31 +1,29 @@
-(identifier) @variable
-
-((constant) @constant
- (#match? @constant "^[A-Z\\d_]+$"))
-
 [
   "alias"
+  "annotation"
   "begin"
   "break"
   "case"
-  "class"
-  "def"
   "do"
   "end"
   "ensure"
+  "extend"
   "in"
-  "module"
+  "include"
+  "lib"
   "next"
-  "rescue"
+  "select"
   "then"
-  "unless"
-  "until"
   "when"
-  "while"
 ] @keyword
```

Comment:
`type` should be a keyword, but only inside `lib_def`

Reply:
Since the "type" string is just used on lib_def, just adding "type" on `@keyword.type` works to highlight it only on lib_def. I did this on last PR update.